core-bundle/src/Resources/contao/library/Contao/Database/Statement.php

```diff
@@ -197,7 +197,8 @@ public function limit($intRows, $intOffset=0)
 	{
 		if ($intRows <= 0)
 		{
-			$intRows = 30;
+			// Ensure offset works if > 0 and limit is set to 0
+			$intRows = PHP_INT_MAX;
```

Comment:
```suggestion
			$intRows = '18446744073709551615';
```

Should we use this hard coded value instead after all, just like in [`doctrine/dbal`](https://github.com/doctrine/dbal/blob/7a8252418689feb860ea8dfeab66d64a56a64df8/src/Platforms/AbstractMySQLPlatform.php#L55)?

Reply:
Fine with me :)